piperider_cli/profiler/profiler.py

```diff
@@ -411,37 +459,43 @@ def _profile_table(self, name: str, table: Table) -> dict:
 
     def _profile_column(self, table_name, table: Table, column: Column) -> dict:
         profiler_config = self.config.profiler_config if self.config else {}
-        if isinstance(column.type, String):
+        column_type = column.type
+        schema_type = str(column.type)
+        if isinstance(column_type, ARRAY) and column_type.item_type is not None:
+            column_type = column_type.item_type
+            schema_type = f"ARRAY<{column_type}>"
+
+        if isinstance(column_type, String):
             # VARCHAR
             # CHAR
             # TEXT
             # CLOB
             generic_type = "string"
             profiler = StringColumnProfiler(self.engine, profiler_config, table, column)
-        elif isinstance(column.type, Integer):
+        elif isinstance(column_type, Integer):
             # INTEGER
             # BIGINT
             # SMALLINT
             generic_type = "integer"
             profiler = NumericColumnProfiler(self.engine, profiler_config, table, column, is_integer=True)
-        elif isinstance(column.type, Numeric):
+        elif isinstance(column_type, Numeric):
             # NUMERIC
             # DECIMAL
             # FLOAT
             generic_type = "numeric"
             profiler = NumericColumnProfiler(self.engine, profiler_config, table, column, is_integer=False)
-        elif isinstance(column.type, Date) or isinstance(column.type, DateTime) or \
+        elif isinstance(column_type, Date) or isinstance(column.type, DateTime) or \
```

Comment:
`isinstance(column.type, DateTime)` should be `isinstance(column_type, DateTime)`